tests/components/harmony/test_config_flow.py

```diff
@@ -16,23 +16,6 @@ def _get_mock_harmonyapi(connect=None, close=None):
     return harmonyapi_mock
 
 
-def _get_mock_harmonyclient():
-    harmonyclient_mock = MagicMock()
```

Comment:
I've transitioned most of this to an actual fake in conftest.py - this allows many more tests to be written that deal with state changes and state management.